spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -109,7 +115,7 @@
       end
 
       context "with a provider" do
-        let(:provider) { FactoryGirl.create(:provider_embedded_ansible, :with_authentication) }
+        let!(:provider) { FactoryGirl.create(:provider_embedded_ansible, :with_authentication) }
```

Comment:
This was added because all examples were already calling `provider` and my new example wasn't, but still needed it created.